uaclient/tests/test_ua_timer.py

```diff
@@ -55,22 +56,37 @@ def test_run_jobs_persists_job_status_on_successful_run(
                 {
                     "update_messaging": {
                         "next_run": now - datetime.timedelta(seconds=1)
-                    }
+                    },
+                    "update_status": {
+                        "next_run": now - datetime.timedelta(seconds=1)
+                    },
                 },
             )
         run_jobs(cfg=cfg, current_time=now)
-        next_run = now + datetime.timedelta(seconds=43200)
+        update_messaging_next_run = now + datetime.timedelta(seconds=43200)
```

Comment:
This time value should be from a const or config right so that it matches up with the UACLIENT_JOBS definition?